### PR TITLE
FALCON-2010 Fix UT errors due to ActiveMQ upgrade

### PR DESCRIPTION
**Pull request by yzheng-hortonworks**
Fixed two test failures due to the upgrade:

1. JMSMessageConsumerTest
After upgrade, some messages are consumed faster and so the initial expectation on no message being dequeued right after the message sending doesn't hold. Also confirmed with @bvellanki that this kind of tests are flaky and have caused intermittent test errors before. Therefore, removed getTotalDequeueCount assertion. In addition, the expected message consumptions are tested in existing code "Mockito.verify(jobEndService,...".

2. FalconPostProcessingTest
After upgrade, Falcon will run into an error if it tries to delete messages when ActiveMQ service is stopped. Therefore, need to call start before deleting the message during teardown.

### CODE REVIEW

File: messaging/src/test/java/org/apache/falcon/messaging/JMSMessageConsumerTest.java [hunk 1]
```diff
@@ -309,7 +298,6 @@ public void testJMSMessagesFromOozieForNonFalconWF() throws Exception {
         sendMessages(TOPIC_NAME, WorkflowExecutionContext.Type.WORKFLOW_JOB, false /* isFalconWF */);
 
         final BrokerView adminView = broker.getAdminView();
-
         Assert.assertEquals(adminView.getTotalDequeueCount(), 0);
```
Comment (pallavi-rao) [inline messaging/src/test/java/org/apache/falcon/messaging/JMSMessageConsumerTest.java]: May be remove it here too to be consistent and also so the test doesn't fail later.

**Comment by pallavi-rao** [comment]
+1 for the patch other than the minor comment.

File: messaging/src/test/java/org/apache/falcon/messaging/JMSMessageConsumerTest.java (hunk 1 above)
Comment (yzheng-hortonworks) [inline messaging/src/test/java/org/apache/falcon/messaging/JMSMessageConsumerTest.java]: Removed and keep only consumer counts. Thanks for your review, @pallavi-rao !

**Comment by bvellanki** [comment]
+1

**Pull request closed without merging**